install.sh

```diff
@@ -3947,7 +3958,11 @@ install_into_existing_repositories() {
 
     git config --global githooks.previous.searchdir "$RAW_START_DIR"
 
-    LOCAL_REPOSITORY_LIST=$(find "$START_DIR" -type d -name .git 2>/dev/null)
+    LOCAL_REPOSITORY_LIST=$(
+        find "$START_DIR" \( -type d -and -name .git \) -or \
+            \( -type f -and -name HEAD -and -not -path "*/.git/*" \) 2>/dev/null
```

Comment:
`-type f -and -name HEAD -and -not -path "*/.git/*"` that's a bit of a weak test. Can we do better to find likely repositories but discard false positives?

Reply:
I could not come up with a better thing. We would need a file in the main folder of the repo:
There is a `continue` in the loop below...
```
HEAD
hooks
objects
refs
```